import/popups/BlockedPopupDialog.qml

```diff
@@ -0,0 +1,63 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Dialog {
+    id: dialog
+
+    property alias host: blockedPopup.host
+    property alias rememberValue: blockedPopup.rememberValue
+
+    BlockedPopupInterface {
+        id: blockedPopup
+
+        anchors.fill: parent
+        rememberValue: remember.checked
+
+        Column {
+            width: parent.width
+            spacing: Theme.paddingMedium
+
+            DialogHeader {
+                //: Allow the site to open a popup window
+                //% "Allow"
+                acceptText: qsTrId("sailfish_components_webview_popups-he-load_popup")
+
+                //: Deny the site from opening a popup window
+                //% "Deny"
+                cancelText: qsTrId("sailfish_components_webview_popups-he-block_popup")
+            }
+
+            Label {
+                x: Theme.horizontalPageMargin
+                width: parent.width - x * 2
+
+                //: %1 is the site that wants to load a popup
+                //% "The site %1 attempted to open a new tab, which has been blocked. Would you like to allow it to open new tabs?"
```

Comment:
Currently the `host` value is actually the url of the page being loaded rather than the host. It might be worth updating the wording to "page" instead of "site", or alternatively getting `EmbedLiteChromeManager.js` to pass the host instead.

It might also be worth considering truncating the value to some fixed length (since the page isn't scrollable, a really long domain or url will cause the "remember" button to get lost).

Reply:
changed to "page".  The "host vs page url" discussion is important, but slightly out of scope for this task/PR, I think.

Reply:
Regarding the truncation: what it is a good way to do this, considering that it should scale appropriately for different screen sizes and form factors?